apps/core/helpers.py

```diff
@@ -23,12 +23,13 @@ def create_i18n_page(content, is_homepage=False, **kwargs):
     languages = list(content.keys())  # convert `dict_keys` to list so it can be poped
     first_language = languages.pop(0)
     slug = slugify(content[first_language])
+    reverse_id = kwargs.pop("reverse_id", slug)
```

Comment:
I think you should do kwargs.get("reverse_id", slug) and not mutate kwargs?

Reply:
`reverse_id` would then be passed 2 time to the function. as an argument and as kwargs